libs/contract-core/seed/devlive/index.ts

```diff
@@ -0,0 +1,56 @@
+import { DateTime } from 'luxon';
+import { log } from '@thecointech/logging';
+import { ConnectContract, TheCoin, COIN_EXP } from '@thecointech/contract-core';
```

Comment:
Moved seeding out of migrations, because it uses a different HW wallet to the original, and we can't do both at the same time